### PR TITLE
[WIP][CR] Newbie: Added Some Parks and a House.

### PR DESCRIPTION
**Pull request by RogueYun**
I added five parks and modified the old basketball court to have more correct lines.

The five parks are these.
~Two scenic/walking parks.
~Tennis court.
~Volleyball court.
~Dog park.

The house was just practice to try my hand at making something but I think it turned out looking very nice.

Let's hope I don't break anything :(

### CODE REVIEW

File: data/json/mapgen/park.json [hunk 1]
```diff
@@ -76,48 +97,363 @@
...
+				"                        "
             ],
             "terrain": {
                 ".": "t_pavement_y",
                 "7": "t_backboard",
                 "|": "t_chainfence_v",
                 "-": "t_chainfence_h",
-                "+": "t_chaingate_l"
+                "+": "t_chaingate_c",
+				"H": "t_pavement",
```
Comment (Coolthulhu) [inline data/json/mapgen/park.json]: You've got tabs here. We use spaces to indent stuff.

File: data/json/mapgen/house_garage6.json [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+				"D": "f_desk",
+				"R": "f_trashcan",
+				"E": "f_cupboard",
+				"F": "f_fridge",
+				"O": "f_oven",
+				"L": "f_locker",
+				"H": "f_armchair",
+				"P": "f_sign"
+			},
+			"toilets": {
```
Comment (Coolthulhu) [inline data/json/mapgen/house_garage6.json]: Remove the section if it's empty.
But should it really be empty? A house with two bedrooms but no toilet?

File: data/json/mapgen/park.json (hunk 1 above)
Comment (RogueYun) [inline data/json/mapgen/park.json]: I beg your pardon, I'm completely new to this. Should I make and commit the suggested changes and do another pull request? Or is there another way I should go about resolving these?

Comment (Rivet-the-Zombie) [inline data/json/mapgen/park.json]: Make the requested changes then commit/push.

File: data/json/mapgen/house_garage6.json (hunk 2 above)
Comment (RogueYun) [inline data/json/mapgen/house_garage6.json]: I appreciate the feedback, It is certainly welcome. But, I'm afraid I don't follow this. What empty section exactly? Do you mean the empty curly brackets after "toilets": {";":? The toilets seem to spawn fine with those brackets being empty. I just copy pasted this section from one of the other house.json files, most have this coding for toilets.

Comment (Coolthulhu) [inline data/json/mapgen/house_garage6.json]: Oops, my fault.

Toilets section is usually for filling toilets with water. If they work fine (ie. have water), then I guess it must be some sort of a hardcoded interaction.

Comment (BevapDin) [inline data/json/mapgen/house_garage6.json]: The toilet section could contain the amount of water to put into the toilet, but that is optional and an empty JSON object is just fine. It's already used that way in most mapgens.

**Comment by vache** [comment]
I think we can consider this one no longer WIP and ready to merge?
